api/src/main/kotlin/com/ford/internalprojects/peoplemover/E2ETestController.kt

```diff
@@ -29,8 +32,14 @@ class E2ETestController(
     }
 
     @DeleteMapping("/api/reset/{uuid}")
-    fun deleteTestSpace(@PathVariable uuid: String): ResponseEntity<Unit> {
-        spaceService.deleteSpace(uuid)
+    fun deleteTestSpaces(@PathVariable uuid: String): ResponseEntity<Unit> {
+        spaceRepository.deleteByUuid(uuid)
+
+        userSpaceMappingRepository.findAllByUserId("USER_ID").let { userSpaceMappings ->
+            for (space in userSpaceMappings) {
+                spaceRepository.deleteByUuid(space.spaceUuid)
+            }
+        }
         localDataGenerator.resetSpace(uuid)
         return ResponseEntity.ok()
                 .header("Set-Cookie", "accessToken=${createJWT()}")
```

Comment:
This test cleanup function is kinda meh, cuz it's not actually deleting any of the people, assignments, products, etc, so it can't really be ran against a database that you don't want to clutter anyway.. Idk if we want to just clear all the tables instead of this logic.

Reply:
I think that if it is on a real database (set up with flyway), then deleting the space automatically deletes all of the entities attached to the space (projects, people, assignments, etc). 
If it isn't attached to a real database, then it probably doesn't matter, as it is probably a test instance.

Reply:
Ah. Gotcha. That makes sense then